xmtp_mls/src/groups/message_history.rs

```diff
@@ -78,9 +105,31 @@ impl<ApiClient> Client<ApiClient>
 where
     ApiClient: XmtpApi,
 {
+    pub fn get_sync_group(&self) -> Result<(Vec<u8>, MlsGroup), GroupError> {
+        let conn = self.store().conn()?;
+        let sync_group_id = conn
+            .find_sync_groups()?
+            .pop()
+            .ok_or(GroupError::GroupNotFound)?
+            .id;
+        let sync_group = self.group(sync_group_id.clone())?;
+
+        Ok((sync_group_id, sync_group))
+    }
+
     pub async fn allow_history_sync(&self) -> Result<(), GroupError> {
```

Comment:
It would be helpful to have a comment explaining what this function actually is allowing. Or maybe renaming it.

All the function does is get or create a sync group, and syncs it. But I'm assuming this is opting me in to respond to message history requests, right?

Reply:
good point, i think `enable_history_sync` is a better name. we could also just get rid of this function altogether and incorporate the creation of a sync group in the `get_sync_group` fn.